src/context/ApplicationContext.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState, useContext, createContext, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { UrlSearchParamBuilder } from '../utils/api-utils/UrlSearchParamBuilder';
+import { getTimePeriodTypes } from '../api/services/timecardService';
+
+export const ApplicationContext = createContext();
+
+const fetchTimePeriodTypesData = async () => {
+  try {
+    const params = new UrlSearchParamBuilder()
+      .setTenantId('00000000-0000-0000-0000-000000000000')
+      .getUrlSearchParams();
+    const response = await getTimePeriodTypes(params);
+    if (response.data?.items) {
+      return response.data.items;
+    }
```

Comment:
(warning: pedantic comment) I think we can make this one line and it’ll have the same behaviour?
```suggestion
    return response.data?.items;
```

Reply:
We can, I've done it